src/app/activity/bounty/Create.ts

```diff
@@ -1,145 +1,187 @@
+            createRequest,
+            null,
+            null,
+            null,
+            guildMember,
+            owedTo);
+    }
 
-    const newBounty = await createDbHandler(
-        createRequest,
-        description,
-        criteria,
-        dueAt,
-        guildMember);
+    Log.info(`user ${guildMember.user.tag} inserted bounty into db`);
 
+    let bountyCard: MessageOptions;
 
-    Log.info(`user ${guildMember.user.tag} inserted bounty into db`);
-    let bountyPreview: MessageOptions = {
-        embeds: [{
-            title: await BountyUtils.createPublicTitle(newBounty),
-            url: (process.env.BOUNTY_BOARD_URL + newBounty._id),
-            author: {
-                icon_url: guildMember.user.avatarURL(),
-                name: `${newBounty.createdBy.discordHandle}: ${guildId}`,
-            },
-            description: newBounty.description,
-            fields: [
-                // TODO: figure out a way to explicitly match order with BountyEmbedFields
-                // static bountyId = 0;
-                // static criteria = 1;
-                // static reward = 2;
-                // static status = 3;
-                // static deadline = 4;
-                // static createdBy = 5;
-                { name: 'Bounty Id', value: newBounty._id.toString(), inline: false },
-                { name: 'Criteria', value: newBounty.criteria.toString() },
-                { name: 'Reward', value: newBounty.reward.amount + ' ' + newBounty.reward.currency, inline: true },
-                { name: 'Status', value: BountyStatus.open, inline: true },
-                { name: 'Deadline', value: BountyUtils.formatDisplayDate(newBounty.dueAt), inline: true },
-                { name: 'Created by', value: newBounty.createdBy.discordHandle.toString(), inline: true },
-            ],
-            timestamp: new Date().getTime(),
-            footer: {
-                text: '👍 - publish | ❌ - delete | Please reply within 60 minutes',
-            },
-        }],
-    };
+    if (createRequest.isIOU) {
+        let bountyCard: MessageOptions = {
+            embeds: [{
+                title: await BountyUtils.createPublicTitle(newBounty),
+                url: (process.env.BOUNTY_BOARD_URL + newBounty._id),
+                author: {
+                    icon_url: guildMember.user.avatarURL(),
+                    name: `${newBounty.createdBy.discordHandle}: ${guildId}`,
+                },
+                description: newBounty.description,
+                fields: [
+                    { name: 'IOU Id', value: newBounty._id.toString(), inline: false },
+                    { name: 'Reward', value: newBounty.reward.amount + ' ' + newBounty.reward.currency, inline: true },
+                    { name: 'Status', value: BountyStatus.open, inline: true },
+                ],
+                timestamp: new Date().getTime(),
+                footer: {
+                    text: '💰 - paid | ❌ - delete ',
```

Comment:
I think we want an action statement here, so maybe `mark as paid` instead of `paid`?

Reply:
Done